.github/workflows/build-and-release.yml

```
@@ -0,0 +1,65 @@
+name: Build and Release
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        go-version: [1.15, 1.16]
+        platform: [amd64, arm64]
```

from PerformanceAnalyst
コードのチェックアウトは通常最初に行うべきです。これにより、他のステップがコードに依存している場合に問題が発生しません。

```suggestion
- name: Check out code
  uses: actions/checkout@v2
```